### ISSUE TITLE
How to build 

### ISSUE THREAD

**Issue by krafi-code** [G-MEGA/TouchPanelWithWebcam#1]
How can I build this project, the project structure  is confusing . Is this project fully open source? Actually I am on Linux. How did you manage latency?

**Comment by G-MEGA** [comment]
This project was made with Unity 2020.
To build this project, open the folder named 'Touch-panel-with-camera' in Unity.

Yeah this project is fully open source.

I didn't manage latency. I just implemented 'judge timing offset' and 'note sync offset'

I'm not good at english, so I don't know if I'm answering correctly.

**Comment by krafi-code** [comment]
괜찮습니다, 제가 조금 한국말 알아요. feels free to use 한국 , the thing is , https://youtu.be/9bXwBRso6gY?si=bDbvnO_InHqzYKDf&t=3m1s  check  this video,  "Make A Laser Security System " in this video he used (Battery ,Ldr (light sensor  , Alarm Buzzer  , Bc547 Transistor ,Laser ,10k Resistor) 

So, what if instend of "Alarm Buzzer" i use a microcontroller to sent keyboard signal to my pc / android ? not here i was thing about latency , I asked chatgpt 

the difference in latency will still be present and generally noticeable. LDRs, being analog sensors, have slower response times compared to a digital switch.
When you use an LDR, it will take time for the Arduino to read the analog value from the LDR, process it, and determine whether a significant change in light level has occurred. This process can introduce a delay in the response time, typically in the range of milliseconds to seconds.

On the other hand, when you use a physical switch with an Arduino, the response is almost instantaneous because the switch sends a digital signal that can be read immediately by the Arduino, resulting in very low latency.

**SO , what do you this is it a loss project ?**

**Comment by krafi-code** [comment]
![IMG_20231022_223933081 MP](https://github.com/G-MEGA/TouchPanelWithWebcam/assets/121858831/b34cbd13-e5e1-4cf4-a739-b94dffdcab39)

**Comment by G-MEGA** [comment]
Umm... sorry, I can't understand what you say.

Do you mean that you want to use arduino to create a sensor for dance rush?

This project use webcams instead of platform and I know nothing about arduino.

**Comment by krafi-code** [comment]
YES, using Laser light to make switch to make dance rush type game  .. just an idea came on my brain today , I just found your video from youtube. by the way nice project .

**Comment by G-MEGA** [comment]
That idea is like a sensor on the platform of dance rush. The difference is that dance rush use infrared light.

When I started this project, I considered same idea too but I gave up because most of people hard to set it up.
However, I think the idea is best way to recreate dance rush.
